deployments/security-groups/allow-to-monitoring.nix

```diff
@@ -0,0 +1,25 @@
+{ globals, ... }:
+
+let
+
+  mkRule = { org, region }: {
+    name = "allow-to-monitoring-${region}-${org}";
+    value = {
+      inherit region;
+      _file = ./allow-to-monitoring.nix;
+      accessKeyId = globals.orgAccessKeys.${org};
+      description = "Access to monitoring server";
+      rules = [{
+	protocol = "tcp";
+	fromPort = 80; toPort = 80;
+	sourceIp = "0.0.0.0/0";
+      } {
+	protocol = "tcp";
+	fromPort = 443; toPort = 443;
```

Comment:
oh, this rule can just be removed
replace it with the existing http(s) rule that also allows 80/443